spec/features/questions/show_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+require 'pry'
+RSpec.describe "display all questions" do
```

Comment:
This test description does not match the test.

Reply:
yeah, fixed.